### PR TITLE
add perfbar v0.2.1 with git autoupdate, close #4814

### LINKED ISSUE #4814: Request to add perfBar
http://lafikl.github.io/perfBar/

### PR DESCRIPTION
**Pull request by maruilian11**
@Amomo could u help me check this PR for #4814? thanks a lot :)
**git repo url:** https://github.com/WPOTools/perfBar
**Watch 31
  Star 1,109
 Fork 43**
* [x] Not found on cdnjs repo
* [x] No already exist issue and PR
* [x] Notable popularity
* [x] Project has public repository on famous online hosting platform (or been hosted on npm)
* [x] Has valid tags for each versions (for git auto-update)
* [x] Auto-update setup
* [x] Auto-update target/source is valid.
* [x] Auto-update filemap is correct.

### CODE REVIEW

**Comment by Amomo** [comment]
@maruilian11 could you add more info about repository & keywords?

**Comment by maruilian11** [comment]
@Amomo i have added the info, couldi u help me check it again? thank you:)

**Comment by Amomo** [comment]
@maruilian11 From the library's bower.json, `"frontend"` and `"performance"` could be keywords also.
Would you mind to add them? It might make user on cdnjs get better search result.

**Comment by maruilian11** [comment]
@Amomo  i have added them, could u help me check it again? thanks.

**Comment by Amomo** [comment]
@PeterDaveHello I think this PR is fine, could you take a look? Thank you!

File: ajax/libs/perfbar/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    "perfbar",
+    "frontend",
+    "performance"
+  ],
+  "autoupdate": {
+    "source": "git",
+    "target": "git://github.com/WPOTools/perfBar.git",
+    "basePath": "lib",
+    "files": [
+      "*"
```
Comment (PeterDaveHello) [inline ajax/libs/perfbar/package.json]: what's the reason why to use wildcard in non-dist folder? We talk about it many times.

**Comment by maruilian11** [comment]
@Amomo sorry, my bad, could u help me check it again? ˊ_>ˋ

**Comment by maruilian11** [comment]
@Amomo is that ok? : D

**Comment by Amomo** [comment]
@maruilian11 good, thank you!
@PeterDaveHello I think this PR is fine. Could you take a look?
Thank you. :-D

**Comment by PeterDaveHello** [comment]
Is classList part of this lib?

**Comment by maruilian11** [comment]
@Amomo could you help me check this PR again? sorry for my bad qualityˊˊ_>ˋ

**Comment by Amomo** [comment]
@maruilian11 It's ok~ We all learn from it. :+1: :+1: 
@PeterDaveHello I think this PR is fine. Could you take a look? Thank you~ :D

**Comment by PeterDaveHello** [comment]
You should confirm is it part of the lib, not remove it because I asked you.

**Pull request merged**